templates/components/common/social-links.html

```diff
@@ -1,19 +1,35 @@
 {{#if social_media}}
     <ul class="socialLinks socialLinks--alt">
         {{#each social_media}}
-            <li class="socialLinks-item">
-                <a class="icon icon--{{display_name}}"
-                   href="{{url}}"
-                   target="_blank"
-                   rel="noopener"
-                   title="{{{capitalize display_name}}}"
-                >
-                    <span class="aria-description--hidden">{{{capitalize display_name}}}</span>
-                    <svg>
-                        <use href="#icon-{{display_name}}"/>
-                    </svg>
-                </a>
-            </li>
+            {{#if display_name '===' 'twitter'}}
+                <li class="socialLinks-item">
+                    <a class="icon icon--x"
+                    href="{{url}}"
+                    target="_blank"
+                    rel="noopener"
+                    title="{{{capitalize display_name}}}"
```

Comment:
@AndrewBarber in this case should we change `title="{{{capitalize display_name}}}"` to just `title="X"`?

Reply:
Agreed, will set static.